diff -Nur a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2020-10-04 11:48:43.000000000 +0200
+++ b/CMakeLists.txt	2021-02-25 10:53:08.953051813 +0100
@@ -17,8 +17,9 @@
 include(KDEFrameworkCompilerSettings NO_POLICY_SCOPE)
 include(KDECMakeSettings)
 
-if (NOT APPLE)
+if (UNIX AND NOT APPLE)
     find_package(X11)
+    find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED X11Extras)
 endif()
 
 set(HAVE_X11 ${X11_FOUND})
