diff -Nur a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2021-02-25 09:36:28.778015205 +0100
+++ b/CMakeLists.txt	2021-02-25 09:40:41.320085809 +0100
@@ -45,6 +45,9 @@
 if (NOT ANDROID)
   find_package(Qt5 ${REQUIRED_QT_VERSION} CONFIG REQUIRED DBus)
 endif()
+if (UNIX AND NOT APPLE)
+  find_package(Qt5X11Extras ${REQUIRED_QT_VERSION} CONFIG)
+endif()
 
 find_package(KF5CoreAddons ${KF5_DEP_VERSION} REQUIRED)
 find_package(KF5ItemViews ${KF5_DEP_VERSION} REQUIRED)
